tests/js/server/aql/aql-optimizer-rule-parallelize-gather-cluster.js

```diff
@@ -108,6 +104,10 @@ function optimizerRuleTestSuite () {
         "FOR doc IN " + cn + " SORT doc.value1 LIMIT 1000 RETURN doc",
         "FOR doc IN " + cn + " SORT doc.value1 LIMIT 1000, 1000 RETURN doc",
         "FOR i IN 1..1000 FOR doc IN " + cn + " FILTER doc.value == i RETURN doc",
+        "FOR doc IN " + cn + " LIMIT 10 UPDATE doc WITH {} IN " + cn,
+        "FOR i IN 1..1000 INSERT {} IN " + cn,
+        "FOR doc1 IN " + cn + " FOR doc2 IN " + cn + " FILTER doc1._key == doc2._key RETURN doc1",
+        "FOR doc1 IN " + cn + " FOR doc2 IN " + cn + " FOR doc3 IN " + cn + " FILTER doc1._key == doc2._key FILTER doc2._key == doc3._key RETURN doc1",
       ];
 
       if (require("internal").options()["query.parallelize-gather-writes"]) {
```

Comment:
This `if` statement seems redundant nowadays, and should always evaluate to `true`. So we may as well remove it now.